src/main/java/com/prgrms/amabnb/common/exception/GlobalExceptionHandler.java

```diff
@@ -43,18 +43,20 @@ public ResponseEntity<ErrorResponse> handleAccessDeniedException(AccessDeniedExc
     @ExceptionHandler(MethodArgumentNotValidException.class)
     public ResponseEntity<ErrorResponse> handleMethodArgumentNotValidException(MethodArgumentNotValidException e) {
         log.error(ERROR_LOG_MESSAGE, e.getClass().getSimpleName(), e.getMessage(), e);
+        ErrorResponse errorResponse = ErrorResponse.of("잘못된 값입니다.", e.getBindingResult());
         return ResponseEntity
             .status(HttpStatus.BAD_REQUEST)
-            .body(new ErrorResponse("Validation 에러가 발생했습니다."));
+            .body(errorResponse);
```

Comment:
이건 다시 쓰이지 않으니 여기 있어도 되지 않을까 생각합니다 ㅎㅎ (optional)
```suggestion
            .body(ErrorResponse.of("잘못된 값입니다.", e.getBindingResult()));
```